src/libraries/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -324,24 +324,14 @@ public void CaseVariantDirectoryNameWithCaseVariantPaths_CaseInsensitiveFileSyst
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.FreeBSD | TestPlatforms.NetBSD)]
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.OSX | TestPlatforms.FreeBSD | TestPlatforms.NetBSD)]
         public void MoveDirectory_FailToMoveDirectoryWithUpperCaseToOtherDirectoryWithLowerCase()
         {
             Directory.CreateDirectory($"{TestDirectory}/FOO");
             Directory.CreateDirectory($"{TestDirectory}/bar/foo");
             Assert.Throws<IOException>(() => Move($"{TestDirectory}/FOO", $"{TestDirectory}/bar/foo"));
         }
 
-        [Fact]
-        [PlatformSpecific(TestPlatforms.OSX)]
-        public void MoveDirectory_NoOpWhenMovingDirectoryWithUpperCaseToOtherDirectoryWithLowerCase()
```

Comment:
I'm not sure why this was removed. Should this throw on Windows & macOS as the source and destinations evaluates identical but the actual directory content can be different? 

Once again this may be beyond intend but it seems like this should be something like `IsCaseInsensitive(TestDirectory)`

Reply:
> I'm not sure why this was removed.

This test fails now. OSX moved to `MoveDirectory_FailToMoveDirectoryWithUpperCaseToOtherDirectoryWithLowerCase`.